gke-disk-image-builder/imager.go

```diff
@@ -29,7 +29,8 @@ import (
 )
 
 const (
-	name = "secondary-disk-image"
+	deviceName = "secondary-disk-image-disk"
```

Comment:
`deviceName` cannot be a constant as it depends on **JobName** (left a related comment below)

Reply:
Not sure if this is the correct understanding, but it reads like this is just the name to reference the attached disk on the VM, and it can be a constant unrelated to JobName? I hardcoded it to "secondary-disk-image-disk" because the `startup.sh` script uses this name.

![image](https://github.com/GoogleCloudPlatform/ai-on-gke/assets/140208917/cd315dc5-8b1e-4734-975b-ddd2696bfab9)

Reply:
Alright, we're good to go I think.
Turned out the device name a constant when you attach the disk to the instance.